nl_server/__init__.py

```diff
@@ -28,6 +29,26 @@ def create_app():
 
   flask_env = os.environ.get('FLASK_ENV')
 
+  # Download existing embeddings (if not already downloaded).
+  models_downloaded_paths = {}
+  models_config_path = '/datacommons/nl/models.yaml'
+  if flask_env in ['local', 'test', 'integration_test', 'webdriver']:
+    models_config_path = os.path.join(
+        os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+        'deploy/nl/models.yaml')
+  app.config['MODELS_CONFIG_PATH'] = models_config_path
+  with open(app.config['MODELS_CONFIG_PATH']) as f:
+    models_map = yaml.full_load(f)
+    if not models_map:
+      logging.error("No configuration found for model")
+      return
+
+    for m in models_map:
+      # Download (only downloads if not already done so).
+      download_path = gcs.download_model_folder(gcs.local_folder(),
+                                                models_map[m])
+      models_downloaded_paths[models_map[m]] = download_path
```

Comment:
nit: can this be a helper in loader?

`models_downloaded_paths = loader.download_models(app.config['MODELS_CONFIG_PATH'])`